### DISCUSSION TITLE
花火大人驾到！通通闪开

### DISCUSSION THREAD

**Discussion by NaiLaowww** [share121/inter-knot#2133] [其他]
恭迎花火大人！

<img src="https://bbs-static.miyoushe.com/static/2024/01/31/a065264ccf369d43f39162320393d9bf_5251148779871794599.png" alt="花火 我来啦" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/01/31/f8a6716e3f0ab4d3235fbb8fd50e481f_2873815640813891077.png" alt="花火 点火" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/01/31/cf527198a404a4db566dc4a9d68befc3_2947652818567379105.png" alt="花火 嘿嘿" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/01/31/42b5ef8bab5a58fbdd74f17c21c12420_2665774360840406961.png" alt="花火 略略略" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/06/19/8079e2c46c0698178c735198f9340cff_3596969066154449614.png" alt="花火 礼物" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/06/19/3472710a3b57940c4e56f828fda903af_331180519040296987.png" alt="花火 锤飞" width="75" >

**Comment by chen92525456** [comment]
确实<img src="https://upload-bbs.mihoyo.com/upload/2022/11/11/c9c0e8c58cb0903ae2328287f5f8f967_461278183372971024.png" alt="胡桃-酷" width="75" >

**Comment by skysnown** [comment]
许愿花火不歪
